extensions/html/server/src/modes/pathCompletion.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import { TextDocument, Position, CompletionList, CompletionItemKind } from 'vscode-languageserver-types';
+import { WorkspaceFolder } from 'vscode-languageserver-protocol/lib/protocol.workspaceFolders.proposed';
+import * as path from 'path';
+import * as fs from 'fs';
+import uri from 'vscode-uri';
+
+export function getPathCompletionParticipant(
+	document: TextDocument,
+	position: Position,
+	result: CompletionList,
+	workspaceFolders: WorkspaceFolder[] | undefined
+) {
+	return {
+		onHtmlAttributeValue: (tag, attributeName, attributeValue) => {
+			const pathTagAndAttribute: { [t: string]: string } = {
+				a: 'href',
+				script: 'src',
+				img: 'src',
+				link: 'href'
+			};
+
+			const isDir = (p: string) => fs.statSync(p).isDirectory();
+
+			if (pathTagAndAttribute[tag] && pathTagAndAttribute[tag] === attributeName) {
+				const currPath = attributeValue.replace(/['"]/g, '');
```

Comment:
only remove quotes are start and end, also should decode.

Reply:
Agree with quotes. Don't think we should decode since this is path completion not URI completion. I'll add a guard against `https`, `http`, `//`.